web/public/mgmt/config.yml

```diff
@@ -2714,6 +2719,8 @@ collections:
       - { label: "Url Slug / Filename", name: "urlSlug", widget: "write-once-read-only" }
       - { label: "Name", name: "name", widget: "string" }
       - { label: "Extension", name: "extension", widget: "boolean", required: false }
+      - { label: "Summary Description", name: "summaryDescriptionMd", widget: "markdown", required: false }
+      - { label: "Body Heading (h2 with h3 styling)", name: "bodyHeading", widget: "string", required: false }
```

Comment:
[pebble] What if we made this required? In theory, content may choose not to have a header, but that seems unlikely. I don't know of an instance where we have no header. This could also future-proof us from having this same heading issue again.